internal/storage/activities/delete_amss.go

```diff
@@ -71,18 +59,14 @@ func (a *DeleteFromAMSSLocationActivity) Execute(
 		return &DeleteFromAMSSLocationActivityResult{Deleted: true}, nil
 	}
 
-	// Set up ticker channel if not provided.
-	if a.tickerCh == nil {
-		ticker := time.NewTicker(time.Second * 60)
-		defer ticker.Stop()
-		a.tickerCh = ticker.C
-	}
+	ticker := time.NewTicker(a.pollInterval)
```

Comment:
`time.NewTicker()` will panic if `a.pollInterval` is zero, so I think we should set a minimum value.